### PR TITLE
Rft: reduce docker containers with one dockercompose file

### PR DESCRIPTION
**Pull request by Lvnszn**
reduce docker containers with one dockercompose file.

### CODE REVIEW

File: integrate_test/docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    restart: on-failure
+
+  nacos:
+    image: nacos/nacos-server:1.2.0
+    container_name: nacos-standalone
+    environment:
+      - PREFER_HOST_MODE=hostname
+      - MODE=standalone
+    ports:
+      - "8848:8848"
```
Comment (AlexStocks) [inline integrate_test/docker/docker-compose.yml:17]: zk 的 ports 配置为啥没有引号？

File: integrate_test/docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  zookeeper:
+    image: zookeeper
+    ports:
+      - 2181:2181
```
Comment (AlexStocks) [inline integrate_test/docker/docker-compose.yml:7]: 为何端口这里配置两个？

File: integrate_test/docker/docker-compose.yml (hunk 1 above)
Comment (Lvnszn) [reply integrate_test/docker/docker-compose.yml:17]: addressed

**Review by AlexStocks — APPROVED** [review]

File: integrate_test/dockercompose/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+version: '3'
```
Comment (LaurenceLiZhixin) [inline integrate_test/dockercompose/docker-compose.yml:1]: 还需要增加下etcd的docker

Comment (Lvnszn) [reply integrate_test/dockercompose/docker-compose.yml:1]: addressed

**Review by LaurenceLiZhixin — APPROVED** [review]

**Review by cjphaha — APPROVED** [review]

**Review by zhaoyunxing92 — APPROVED** [review]

**Review by zhaoyunxing92 — APPROVED** [review]

**Pull request merged**